src/NextGen/DonationForm/Actions/ConvertDonationFormBlocksToFieldsApi.php

```diff
@@ -164,8 +164,9 @@ protected function mapGenericBlockAttributesToNode(Node $node, BlockModel $block
                 $node->displayInAdmin = $block->attributes['displayInAdmin'];
             }
 
+            /** TODO: ask kyle about $node->showInReceipt() */
             if($block->hasAttribute('displayInReceipt') && $block->attributes['displayInReceipt']) {
-                $node->displayInAdmin = $block->attributes['displayInReceipt'];
+                $node->displayInReceipt = $block->attributes['displayInReceipt'];
```

Comment:
The macro that I added is `shouldDisplayInReceipt` in order to implement declarative access to the `displayInReceipt` attribute.

```php
Field::macro('shouldDisplayInReceipt', function() {
    return isset($this->displayInReceipt) && $this->displayInReceipt;
});
```

Reply:
It looks like there is a `ShowInReceipt` trait in the Fields API, but it isn't available on the `Field` class. My macro adds it to the `Field` class, but we should refactor the Fields API so that `ShowInReceipt` is added to `Field`.

![image](https://user-images.githubusercontent.com/10858303/211048334-432a9c4d-dafb-4188-8031-cb003b2e288e.png)

Reply:
Yeah, the plan was to circle back an make the change in core, see https://github.com/impress-org/givewp-next-gen/pull/99#discussion_r1038536605.

Reply:
I did actually see this being used on the field already so in theory the code below should technically work:

<img width="1154" alt="Screen Shot 2023-01-06 at 11 14 52 AM" src="https://user-images.githubusercontent.com/10138447/211052797-3ae9a34f-d5cc-4ca5-855e-6e2b5a3368b3.png">


```
$field->showInReceipt($block->attributes['displayInReceipt']);
```